tests/ImageSharp.Tests.ProfilingSandbox/Program.cs

```diff
@@ -32,14 +31,13 @@ private class ConsoleOutput : ITestOutputHelper
         /// </param>
         public static void Main(string[] args)
         {
-            LoadResizeSaveParallelMemoryStress.Run();
```

Comment:
What's happening here? This is how we're currently comparing the memory usage between libraries.

Reply:
I was playing around in the sandbox while doing this PR, most likely it's a side effect, I will roll back to initial state.